### PR TITLE
Update configure-email-notifications.md

### PR DESCRIPTION
**Pull request by doctordns**
Added PowerShell example

### CODE REVIEW

**Comment by PRMerger18** [comment]
@doctordns : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by doctordns** [comment]
any progress?

File: WindowsServerDocs/storage/fsrm/configure-email-notifications.md [hunk 1]
```diff
@@ -31,7 +31,27 @@ To send these notifications and storage reports, you must specify the SMTP serve
 
 6. Click **OK**.
 
+## To configure e-mail options using PowerShell
+
+You use the `Set-FsrmSetting` cmdlet to set the e-mail configuration and the `Send-FsrmTestEmail` cmdlet to sent a test email as follows:
```
Comment (illfated) [inline WindowsServerDocs/storage/fsrm/configure-email-notifications.md:36]: Readability adjustments and typo correction:
```suggestion
You can use the `Set-FsrmSetting` cmdlet to set the e-mail configuration and the `Send-FsrmTestEmail` cmdlet to send a test email as shown in the following example:
```

**Comment by PRMerger9** [comment]
@doctordns : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@illfated: I'm sorry - only the author of this article, @JasonGerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. The sign-off comment is in agreement with the MS Docs team.

**Comment by doctordns** [comment]
Any news?

If there is an issue with the proposed changes, please let me know. Otherwise, can someone accept the PR?

**Comment by illfated** [comment]
@JasonGerend : Any chance you could review this document, or assign someone else to do it instead?

**Comment by sethmanheim** [comment]
Thanks @doctordns, and sorry for the delay. Nothing left to do but smile, smile, smile.

**Pull request merged**